### PR TITLE
argmin #17061

### PR DESCRIPTION
**Pull request by ayam04**
first commit to add the argmin under search.py

### CODE REVIEW

File: ivy/functional/frontends/paddle/tensor/search.py [hunk 1]
```diff
@@ -13,3 +13,7 @@
 @to_ivy_arrays_and_back
 def argmax(x, /, *, axis=None, keepdim=False, dtype="int64", name=None):
     return ivy.argmax(x, axis=axis, keepdims=keepdim, dtype=dtype)
+
```
Comment (Sam-Armstrong) [inline ivy/functional/frontends/paddle/tensor/search.py:16]:
```suggestion


@with_supported_dtypes(
    {"2.4.2 and below": ("float32", "float64", "int16", "int32", "int64", "uint8")},
    "paddle",
)
```

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_search.py [hunk 2]
```diff
@@ -35,3 +35,34 @@ def test_paddle_argmax(
         axis=axis,
         keepdim=keepdim,
     )
+
```
Comment (Sam-Armstrong) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_search.py:38]: Should be 2 lines of whitespace not 1

**Review by Sam-Armstrong — CHANGES_REQUESTED** [review]
@ayam04 Thanks for making the PR! 

The test you wrote is failing at the moment, but I think it should be fixed by the change I requested. You can run the test using the command `pytest ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_search.py::test_paddle_argmin` to check this. 

Also, please could you ensure that there are two lines of whitespace between funtions (both the function and test you added), as this is the standard we are following in Ivy.

Other than these this looks really good, great work! 😊

**Comment by Sam-Armstrong** [comment]
Close https://github.com/unifyai/ivy/issues/17061

**Comment by Sam-Armstrong** [comment]
Forgot to mention, you need to comment 'Close <issue_num>' on any pull requests you make. But I've done it here, so no worries 😊

**Comment by ayam04** [comment]
hey @Sam-Armstrong, I've updated the code accordingly. If you can look into the same.

**Comment by ayam04** [comment]
also if u can look into this https://github.com/unifyai/ivy/pull/16527/files , might helps us.

**Review by Sam-Armstrong — APPROVED** [review]
lgtm! thanks @ayam04

**Comment by ayam04** [comment]
Close #17061

**Pull request merged**